extensions/amp-form/0.1/form-validators.js

```diff
@@ -115,6 +130,16 @@ export class FormValidator {
     return this.form.querySelectorAll('input,select,textarea');
   }
 
+  /**
+   * @return {string} A unique ID.
+   * @protected
+   */
+  createUniqueAriaDescId_() {
+    return (
+      ARIA_DESC_ID_PREFIX + this.uniqueFormId_ + '-' + this.ariaDescCounter_++
```

Comment:
style suggestion: I sometimes prefer template strings, but if you like concatenation better, go for it

```js
return `${ARIA_DESC_ID_PREFIX}${this.uniqueFormId_}-${this.ariaDescCounter_++}`;
```